<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PSMainWin</class>
 <widget class="QMainWindow" name="PSMainWin">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>752</width>
    <height>758</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>PS Remote Control Server</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="SendButton">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>20</y>
      <width>91</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Send</string>
    </property>
   </widget>
   <widget class="QPushButton" name="CloseButton">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>120</y>
      <width>91</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Close</string>
    </property>
   </widget>
   <widget class="QTextBrowser" name="StatusOutput">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>530</y>
      <width>631</width>
      <height>181</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
   </widget>
   <widget class="QPushButton" name="ResetButton">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>70</y>
      <width>91</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Reset</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="PortEdit">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>50</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
    <property name="cursorPosition">
     <number>0</number>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>46</width>
      <height>13</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Port</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="IpEdit">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>20</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>46</width>
      <height>13</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>IP</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="DataEdit">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>130</y>
      <width>511</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>110</y>
      <width>101</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>DataToSend</string>
    </property>
   </widget>
   <widget class="QPushButton" name="StartUpButton">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>20</y>
      <width>91</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>StartUp</string>
    </property>
   </widget>
   <widget class="QPushButton" name="ShutdownButton">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>70</y>
      <width>91</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Shutdown</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="NodeEdit">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>20</y>
      <width>91</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="NodeLabel">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>20</y>
      <width>46</width>
      <height>13</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Node</string>
    </property>
   </widget>
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>190</y>
      <width>591</width>
      <height>301</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>4</number>
    </property>
    <widget class="QWidget" name="SmuTab">
     <attribute name="title">
      <string>SMU</string>
     </attribute>
     <widget class="QPushButton" name="SmuInitPush">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>40</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Init</string>
      </property>
     </widget>
     <widget class="QPushButton" name="SmuResetPush">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>70</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Rest</string>
      </property>
     </widget>
     <widget class="QTextBrowser" name="SmuStatusBrowser">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>130</y>
        <width>511</width>
        <height>121</height>
       </rect>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
     </widget>
     <widget class="QPushButton" name="SmuTestPush">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>40</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>GetVoltage</string>
      </property>
     </widget>
     <widget class="QPushButton" name="SmuTest2Push">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>70</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>GetCurrent</string>
      </property>
     </widget>
     <widget class="QPushButton" name="SetVoltPush">
      <property name="geometry">
       <rect>
        <x>380</x>
        <y>40</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>SetVoltage</string>
      </property>
     </widget>
     <widget class="QPushButton" name="SetCurrPush">
      <property name="geometry">
       <rect>
        <x>470</x>
        <y>40</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>SetCurrent</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="VoltCurrentEdit">
      <property name="geometry">
       <rect>
        <x>380</x>
        <y>10</y>
        <width>161</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>0.1</string>
      </property>
     </widget>
     <widget class="QPushButton" name="SmuCompUPush">
      <property name="geometry">
       <rect>
        <x>380</x>
        <y>70</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>SetVoltComp</string>
      </property>
     </widget>
     <widget class="QPushButton" name="SmuCompIPush">
      <property name="geometry">
       <rect>
        <x>470</x>
        <y>70</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>SetCurrComp</string>
      </property>
     </widget>
     <widget class="QPushButton" name="SmuEnablePush">
      <property name="geometry">
       <rect>
        <x>300</x>
        <y>40</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Enable</string>
      </property>
     </widget>
     <widget class="QPushButton" name="SmuDisablePush">
      <property name="geometry">
       <rect>
        <x>300</x>
        <y>70</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Disable</string>
      </property>
     </widget>
     <widget class="QPushButton" name="SmuSourceIPush">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>40</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>SourceI</string>
      </property>
     </widget>
     <widget class="QPushButton" name="SmuSourceUPush">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>70</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>SourceU</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="RelaisTab">
     <attribute name="title">
      <string>Relais</string>
     </attribute>
     <widget class="QSpinBox" name="ChannelSpin">
      <property name="geometry">
       <rect>
        <x>190</x>
        <y>70</y>
        <width>42</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="RelaisInitPush">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>50</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Init</string>
      </property>
     </widget>
     <widget class="QPushButton" name="RelaisResetPush">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>80</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Rest</string>
      </property>
     </widget>
     <widget class="QTextBrowser" name="RelaisStatusBrowers">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>140</y>
        <width>421</width>
        <height>121</height>
       </rect>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="ServerTab">
     <attribute name="title">
      <string>Server</string>
     </attribute>
    </widget>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>Callib</string>
     </attribute>
     <widget class="QPushButton" name="CalibStartPush">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>20</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Start</string>
      </property>
     </widget>
     <widget class="QPushButton" name="CalibStopPush">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>80</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Stop</string>
      </property>
     </widget>
     <widget class="QTextBrowser" name="CalibStatusBrowser">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>140</y>
        <width>501</width>
        <height>121</height>
       </rect>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
     </widget>
     <widget class="QProgressBar" name="ChProgBar">
      <property name="geometry">
       <rect>
        <x>300</x>
        <y>80</y>
        <width>251</width>
        <height>23</height>
       </rect>
      </property>
      <property name="value">
       <number>1</number>
      </property>
     </widget>
     <widget class="QProgressBar" name="TotProgBar">
      <property name="geometry">
       <rect>
        <x>300</x>
        <y>20</y>
        <width>251</width>
        <height>23</height>
       </rect>
      </property>
      <property name="value">
       <number>1</number>
      </property>
     </widget>
     <widget class="QLineEdit" name="ChStatusLineEdit">
      <property name="geometry">
       <rect>
        <x>300</x>
        <y>50</y>
        <width>113</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="CalWaitPush">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>50</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Wait</string>
      </property>
     </widget>
     <widget class="QPushButton" name="ReloadSettingsPush">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>20</y>
        <width>91</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>ReloadSetting</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>OVP Test</string>
     </attribute>
     <widget class="QPushButton" name="StartOvpTest_PushButton">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>20</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Start Test</string>
      </property>
     </widget>
     <widget class="QPushButton" name="StopOvpTest_PushButton">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>20</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Stop Test</string>
      </property>
     </widget>
     <widget class="QTextBrowser" name="OvpStatusBrowser">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>120</y>
        <width>511</width>
        <height>131</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="InputNode">
      <property name="geometry">
       <rect>
        <x>320</x>
        <y>20</y>
        <width>71</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>250</x>
        <y>20</y>
        <width>47</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:12pt;&quot;&gt;Node&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
     <widget class="QPushButton" name="PrepareGates_PushButton">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>70</y>
        <width>181</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Set SW_SUB and SW_Ref to -15 V</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
